src/wallet/rpcwallet.cpp

```diff
@@ -2444,6 +2449,14 @@ static UniValue getwalletinfo(const JSONRPCRequest& request)
         obj.pushKV("hdseedid", seed_id.GetHex());
     }
     obj.pushKV("private_keys_enabled", !pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS));
+    if (pwallet->IsScanning()) {
+      UniValue scanning(UniValue::VOBJ);
+      scanning.pushKV("duration", pwallet->ScanningDuration() / 1000);
+      scanning.pushKV("progress", pwallet->ScanningProgress());
+      obj.pushKV("scanning", scanning);
+    } else {
+      obj.pushKV("scanning", UniValue());
```

Comment:
```suggestion
        obj.pushKV("scanning", UniValue());
```

https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy

Reply:
Fixed.